src/main/java/io/cdap/plugin/google/common/GoogleDriveBaseConfig.java

```diff
@@ -123,23 +116,18 @@ public String getReferenceName() {
     return referenceName;
   }
 
-  public String getAccessToken() {
-    return accessToken;
-  }
-
   public String getDirectoryIdentifier() {
     return directoryIdentifier;
   }
 
-  public String getClientId() {
-    return clientId;
-  }
-
-  public String getClientSecret() {
-    return clientSecret;
-  }
-
-  public String getRefreshToken() {
-    return refreshToken;
+  public String getAccountFilePath() {
+    if (AUTO_DETECT_VALUE.equals(accountFilePath)) {
+      accountFilePath = System.getenv(GOOGLE_APPLICATION_CREDENTIALS_ENV_PROP);
```

Comment:
I think in my case the library automatically picked up data from the environment variable.
https://github.com/data-integrations/nlp/blob/5c61fb0fc17f010135feaaea4702a114a2a94bc1/nlp-common/src/main/java/io/cdap/google/common/NLPMethodExecutor.java#L81
In your case it does not work like that?

Reply:
You are right, by another way, but it is already implemented, will change it

Reply:
done